index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <main>
+        
+            <nav>
+                <div class="logo">
+                    <p>My Logo</p>
+                </div>
+                
+                <div class="menu-icon">
+                    <img src="images/menu-icon.svg" alt="">
+                </div>
+                
+              
+            </nav>
+        <section class="first-section">
+            <header>
+                <h1>I'm Abigael <br> Glad to see <br> you</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what 
+                    you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </header>
+    
+            <div class="socials">
+                <p>let's connect</p>
```

Comment:
- [ ] Please reduce the font-weight for this text and make sure it matches with the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)design

### Your design

![image](https://user-images.githubusercontent.com/83803414/165054796-ee930a40-d95f-4527-9b0f-4766460e9600.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/165054835-ea3211c0-b543-4a15-b75e-c9a6e29c0054.png)

Reply:
done.